src/terraform/terraform_basics.md

```diff
@@ -46,8 +47,9 @@ provider "grid" {
 
 ## Find your Node
 
-The choice of the node is up to the user. They need to do the capacity planning.
+we have two options here
```

Comment:
We have two options (using the scheduler or searching for a node in the capacity explorer)